api/knexfile.ts

```diff
@@ -0,0 +1,22 @@
+import {typeOf} from "uri-js/dist/esnext/util";
+
+const {knexSnakeCaseMappers} = require('objection');
+require('dotenv').config();
+
+export const boundariesConfig = {
+    client: 'better-sqlite3',
+    connection: {
+        filename: './boundaries.sqlite',
+    },
+    useNullAsDefault: true,
+    pool: {
```

Comment:
Nesu tikras dėl `pool` su `sqlite3`:

https://knexjs.org/guide/#pool
Čia mini kad `and a single connection for sqlite3 (due to issues with utilizing multiple connections on a single file)`

https://github.com/knex/knex/issues/3176
Čia minima `For now I will just follow the knex default and use config.pool.max = 1 for SQLite`

Bet nežinau, jei veikia tai gal ir ne problema,